lib/telegram_step.rb

```diff
@@ -1,26 +1,37 @@
+require 'faraday'
 require 'telegram/bot'
 
 class TelegramStep
+  attr_writer :conn
+
   def initialize(token, chat_id, payload)
     @token = token
     @chat_id = chat_id
     @payload = payload
   end
 
+  def conn
+    @conn ||= Faraday.new
+  end
+
+  private
+
+  attr_reader :token, :chat_id, :payload
+
   def send_message(text)
-    Telegram::Bot::Client.run(@token) do |bot|
```

Comment:
LOL! I though @ were needed in theses places. I'm such a noob at Ruby! Thanks!

Reply:
Well, it's not needed since I added `attr_reader :token, :chat_id, :payload` :-D. The use of @ in your code was right in the term of syntax. I've just removed it for readability, but... it's also a good practice to set instance variables directly, but use getters for reading their values. Even more when you're subclassing.

Thank you for taking your time reviewing this changes!